Week3/StringSegTest.java

```diff
@@ -0,0 +1,47 @@
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.testng.annotations.Test;
+
+
+public class StringSegTest {
+  @Test
+  public void TestBreakable1() {
```

Comment:
It's a good idea to name these tests in a way that lets you know what it is you're testing.  Here are some example naming styles: https://dzone.com/articles/7-popular-unit-test-naming

Reply:
Thank you @mooreaarond  for resource :) I will keep in my mind for future code practice.